### PR TITLE
[FS-16]: Subscriptable Generator

### PR DESCRIPTION
**Pull request by gleandroj**
## Description

[FS-16]: Subscriptable Generator

## Tickets

[FS-16](https://flatirons.atlassian.net/browse/FS-16)

## Screenshots / Demos

[FS-16]: https://flatirons.atlassian.net/browse/FS-16?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lib/flatirons/saas/engine.rb [hunk 1]
```diff
@@ -3,8 +3,11 @@
 require 'money-rails'
 require 'devise'
 require 'rswag'
+require 'rails/generators'
```
Comment (christianpaez) [inline lib/flatirons/saas/engine.rb]: do we need to require all rails generators?
I think active record is enough

**Review by christianpaez — APPROVED** [review]
lgtm

File: spec/generator/flatirons/saas/subscriptable_generator_spec.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+require 'rails_helper'
+require 'generator_spec'
+
+module Flatirons::Saas::Generators
+  RSpec.describe SubscriptableGenerator, type: :generator do
+    destination File.expand_path('../../../../tmp', __dir__)
+    arguments %w(user_test)
+
+    before(:all) do
+      allow(Rails::VERSION::MAJOR) { 6 }
```
Comment (mikefrederick) [inline spec/generator/flatirons/saas/subscriptable_generator_spec.rb]: 7

**Review by mikefrederick — CHANGES_REQUESTED** [review]

File: spec/generator/flatirons/saas/subscriptable_generator_spec.rb (hunk 2 above)
Comment (gleandroj) [reply spec/generator/flatirons/saas/subscriptable_generator_spec.rb]: It is just for test propose

File: lib/flatirons/saas/engine.rb (hunk 1 above)
Comment (gleandroj) [reply lib/flatirons/saas/engine.rb]: Removed now

File: lib/generators/active_record_base_generator.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class ActiveRecordBase < ::ActiveRecord::Generators::Base
+  include ::Rails::Generators::Migration
+
+  def migration_name
```
Comment (christianpaez) [inline lib/generators/active_record_base_generator.rb]: I think this in defined in Rails::Generators::Migration

**Review by christianpaez — APPROVED** [review]
lgtm

**Pull request merged**